app/javascript/src/Logger.js

```diff
@@ -0,0 +1,34 @@
+import { Logtail } from '@logtail/browser';
+
+export default class Logger {
+  #environment;
+  #gameId;
+
+  #logtail;
+
+  constructor(environment, gameId) {
+    this.#environment = environment;
+    this.#gameId = gameId;
+
+    this.#logtail = new Logtail('3bdHcA8P3mcww2ojgC5G8YiT');
+  }
+
+  error(message, data) {
+    data.gameId = this.#gameId;
+
+    switch (this.#environment) {
+      case 'production':
+        this.#logtail.error(message, data);
+        break;
+
+      case 'replay':
```

Comment:
Should we populate this with `this.#logtail.error(message, data);` like we do in `production`?

Reply:
Im not sure, wouldn't that mean that we get a duplicate `logtail.error` everytime someone replays the game?

Reply:
I think not because the instance of `Logger` after they've clicked the "back" button (or similar) would have the `replay` scope and not the `production` scope, right? I think the code as written here will swallow the error if they are rewinding the game.

Reply:
Is the goal to suppress logs from replay? If so, then this is fine as-is. But if we want logs from replay then we want it to act like production above, right? I'm ok either way, though. 😄